### PR TITLE
[Feature] Parallel Jobs 

### PR DESCRIPTION
**Pull request by Quizenger**

### CODE REVIEW

**Review by NathanTP — COMMENTED** [review]
jobs test is failing:
```
(fs) [xarc@xarc0 jobs]$ python test.py ../../marshal
Cleaning the test the first time:
To check on progress, either call marshal with '-v' or see the live output at:
/data/repos/chipyard/software/firemarshal/logs/jobs-clean-2021-12-27--23-53-50-8DBNZOHE3MJHPYS8.log
Log available at: /data/repos/chipyard/software/firemarshal/logs/jobs-clean-2021-12-27--23-53-50-8DBNZOHE3MJHPYS8.log
Build Workload
Run Root Only
[screen is terminating]
Expected output 'root : run' not in stdout
Failure: contained incorrect outputs
```

File: docs/source/Tutorials/paralleljobs.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+The same login details and exit procedure apply.
+
+Note that both the root and the job workloads can't be launched at once. 
+
+Let us now launch both jobs in this workload using:
+
+::
+
+  ./marshal launch --all example-workloads/parallelJobs.yaml
```
[the remark this answers is not in the document]
Comment (Quizenger) [reply docs/source/Tutorials/paralleljobs.rst:52]: Resolved!

File: wlutil/launch.py [hunk 2]
```diff
@@ -100,34 +115,56 @@ def launchWorkload(baseConfig, jobs=None, spike=False, interactive=True):
...
+
+    try:
+        for config in configs:
+            if config['launch']:
+                runResDir = baseResDir / config['name']
+                uartLog = runResDir / "uartlog"
+                os.makedirs(runResDir)
+
+                if spike:
+                    if 'img' in config and not config['nodisk']:
```
Comment (NathanTP) [inline wlutil/launch.py:129]: You'll have a merge conflict with a recent change I pushed (#224). You can look at the change I made to launch.py there and just apply the same concept here.

**Review by NathanTP — COMMENTED** [review]
This looks great. Just fix the merge conflict and we should be good to go.

**Review by NathanTP — APPROVED** [review]
Looks good. I ran the tutorial and the unit test. I ran fullTest.py on an earlier commit and I assume you did more recently.

**Pull request merged**